Gopkg.lock

```diff
@@ -384,7 +384,7 @@
 
```

Comment:
haven we have a seprate commit for go dep changes

Reply:
Resolved